style.css

```diff
@@ -0,0 +1,34 @@
+body {
+    padding: 10%;
+}
+
+h1 {
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 400 !important;
+}
+
+p {
+    margin: 0px;
+    margin-right: 10px;
+}
+
+.btn {
+    margin-left: 10px;
+}
+
+#fibo_text {
```

Comment:
keep with the same naming style across all of the css selector. consider using #fibo-text intext of #fibo_text